packages/hop-node/src/cli/hopNode.ts

```diff
@@ -41,13 +41,16 @@ program
   )
   .option('--clear-db', 'Clear cache database on start')
   .option('--log-db-state', 'Log db state periodically')
+  .option('--sync-from-date <string>', 'Date to start syncing db from, in format YYYY-MM-DD')
   .action(async (source: any) => {
     try {
       printHopArt()
 
       const configFilePath = source.config || source.args[0]
       const config: FileConfig = await parseConfigFile(configFilePath)
       await setGlobalConfigFromConfigFile(config, source.passwordFile)
+      const syncFromDate = source.syncFromDate
```

Comment:
what date format should this be?

Reply:
ISO format,
example in cli flag help

![DeepinScreenshot_select-area_20210901184510](https://user-images.githubusercontent.com/168240/131768118-c8e48a86-b5e9-441d-addf-de031e19ff3d.png)